engine.json

```diff
@@ -7,5 +7,5 @@
   },
   "languages" : ["TypeScript"],
   "version": "51e260eb5c8842431dad261fa70a700382cf9e60",
-  "spec_version": "0.2.3"
+  "spec_version": "0.3.1"
```

Comment:
Did anything else in this PR change the spec compatibility? If not, this bump could probably be its own commit.

Reply:
`engine.json` has been moved from `/usr/src/app` to `/`.

Reply:
Ahh, awesome 👍. Still think that might be nice to specify in a separate commit.